kafka-connect-influxdb/src/main/scala/com/datamountaineer/streamreactor/connect/influx/helpers/Util.scala

```diff
@@ -0,0 +1,21 @@
+package com.datamountaineer.streamreactor.connect.influx.helpers
+
+import scala.util.{Failure, Success, Try}
+
+object Util {
+
+  def shortCircuitOnFailure[T](a: Try[Seq[T]], b: Try[T]): Try[Seq[T]] = (a, b) match {
+    case (Failure(e), _) => Failure(e)
+    case (Success(r), Success(value)) => Success(r :+ value)
+    case (_, Failure(e)) => Failure(e)
+  }
+
+  def caseInsensitiveComparison(a: String, b: String): Boolean = a.toUpperCase == b.toUpperCase
+
+  def caseInsensitiveComparison(a: Seq[String], b: Seq[String]): Boolean = a.map(_.toUpperCase) == b.map(_.toUpperCase)
+
+  val KEY_CONSTANT = "_key"
```

Comment:
For now I believe this can live here but it would be nice to move this out of this project and into the parser side of things